File: OAIP/LR-3-4/2 лаба 4.1.pas
program aa;
var
  n, i: integer;
  besttime: real;
  currenttime: real;
  bestswimmer: string;
begin
  readln(n);
  besttime := maxint; 
  for i := 1 to N do
  begin
    readln(bestswimmer); 
    readln(currenttime); 
    if currenttime < besttime then 
    begin
      besttime := currenttime; 
      bestswimmer := bestswimmer; 
    end;
  end;
  writeln(bestswimmer, ' ', besttime:0:2); 
end.

File: OAIP/LR-3-4/2 лаба 4.1.py
n = int(input())
besttime = float('inf')
bestswimmer = ""
for i in range(n):
    bestswimmer = input().strip()
    currenttime = float(input().strip())
    if currenttime < besttime:
        besttime = currenttime
        bestswimmer = bestswimmer
print(f"{bestswimmer} {besttime:.2f}")